docs/logs/getting-started/README.md

```diff
@@ -1,3 +1,86 @@
-# Getting Started with OpenTelemetry .NET in 5 Minutes
+# Getting Started with OpenTelemetry .NET Logs in 5 Minutes
 
-TBD
+First, download and install the [.NET Core
+SDK](https://dotnet.microsoft.com/download) on your computer.
+
+Create a new console application and run it:
+
+```sh
+dotnet new console --output getting-started
+cd getting-started
+dotnet run
+```
+
+You should see the following output:
+
+```text
+Hello World!
+```
+
+Let's install the packages to get started with logging. If your ConsoleApp
+targets `netcoreapp2.1` install the version `2.1.1` of
+`Microsoft.Extensions.Logging` and `Microsoft.Extensions.DependencyInjection`
+packages:
+
+  ```sh
+  dotnet add package Microsoft.Extensions.Logging -v 2.1.1
+  ```
+
+  ```sh
+  dotnet add package Microsoft.Extensions.DependencyInjection -v 2.1.1
+  ```
+
+If your ConsoleApp targets one of these frameworks:
+`net461`,`net462`,`net47`,`net471`,`net472`,`net48`,`netcoreapp3.1`, install the
```

Comment:
you can simply say, for anything other than 2.1, instead of explicitly listing all framewkros.

Reply:
Fixed this.